src/js/pensions/config/form.js

```diff
@@ -113,6 +114,24 @@ function usingDirectDeposit(formData) {
   return formData['view:noDirectDeposit'] !== true;
 }
 
+function servedDuringWartime(period) {
+  const warDates = [
+    [-1692982800000, -1664384400000], // Mexican Border Period (May 9, 1916 - April 5, 1917)
```

Comment:
Calculations for reference:
<img width="385" alt="screen shot 2017-07-13 at 4 45 34 pm" src="https://user-images.githubusercontent.com/16051172/28192404-ce3d46ae-67ea-11e7-925d-a81c879d19ee.png">